autorest/codegen/serializers/__init__.py

```diff
@@ -150,7 +150,7 @@ def _serialize_and_write_operations_folder_process(
         filename: str
     ) -> None:
         # write sync operation group and operation files
-        if not code_model.options['python_3_only'] and not code_model.options["add_python_3_operation_files"]:
+        if not code_model.options['python_3_only']:
```

Comment:
when `version_tolerant` is `True`, `add_python_3_operation_files` is `True`. But we also need `_operations.py` to support py2.7 until next year. So we delete judgement for `add_python_3_operation_files`

Reply:
sorry thought i left a comment here last week, reposting again. Basically, this code won't work if someone doesn't pass in flag `--python3-only`, but passes in `add-python3-operation-files`, which is something we really don't want. Even though their defaults depend on each other, these are 2 different flags, and we want them to be able to work independently

To further clarify on your comment:  `version-tolerant` and `add-python3-operation-files` are 2 different flags. Yes, when `version-toleratn` is `True`, the default for `add-python3-operation-files` is `True`, but if users explicitly passed in `add-python3-operation-files` as `False`, we would have `version-tolerant` `True` and `add-python3-operation-files` as `False`.

Reply:
Hi @iscai-msft The judge is for `_operations.py` while `add-python3-operation-files` is for `_operations_py3.py`. So no matter  whether users set `add-python3-operation-files` or what default value of `add-python3-operation-files` is, `add-python3-operation-files` should not exist here.

Reply:
ok i think I'm seeing what you're saying more, sorry for the confusion!